agent/bench-scripts/tests/pbench-specjbb2005/test-56.pre

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+
+_dir="${_testtmp}/specjbb2005"
+mkdir ${_dir} || exit 1
+touch ${_dir}/SPECjbb.props || exit 1
+printf "input.jvm_instances=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.measurement_seconds=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.starting_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.increment_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
+printf "input.ending_number_warehouses=_replace_\n" >> ${_dir}/SPECjbb.props
```

Comment:
I missed it:  where does the `_replace_` token get replaced?

Reply:
Line 279 to 285 in `pbench-specjbb2005` ... it generates the `.sed` file you commented on elsewhere to patch command line parameters into the template `.props` file. This is an ... interesting ... implementation.

Reply:
Thanks, Dave...so, the mechanism has long been there, we're just adding a new engagement of it?  OK, that's cool.